starlark/testdata/bytes.star

```diff
@@ -0,0 +1,131 @@
+# Tests of 'bytes' (immutable byte strings).
```

Comment:
Do we have a test somewhere that you can't assign to `b[i]`? Same for strings.

Reply:
Added.